src/config.cpp

```diff
@@ -221,9 +221,11 @@ void parse_program_options(int argc, const char** argv)
         .metavar("NAME")
         .optional();
 
+    system_mode_options.toggle("system-wide",
```

Comment:
This option seems confusing. What you want to use is a [reversable toggle](https://github.com/tud-zih-energy/nitro/blob/master/tests/options_test.cpp#L412) called `process-recording` (or something like that) with a default of `true`.

So you'd call lo2s with `.../lo2s -a --no-process-monitoring [...]`.

Reply:
One of the grievances I have with the lo2s CLI is that it is not very orthogonal in places.

However here consistency is probably better than trying to shoehorn in orthogonality...